tools/oatmeal/dump-oat.cpp

```diff
@@ -2248,7 +2248,8 @@ class OatFile_124 : public OatFile {
     auto key_value_store = KeyValueStore(
         buf.slice(header.size()).truncate(header.key_value_store_size));
 
-    auto rest = buf.slice(header.size() + header.key_value_store_size);
+    // auto rest = buf.slice(header.size() + header.key_value_store_size);
```

Comment:
I'd prefer to not leave commented out code behind. Version control saves this for us. We don't need to comment it out.